features/create_membership_application.feature

```diff
@@ -27,10 +27,14 @@ Feature: Create a new membership application
       | admin@shf.se           | yes   |        |            |           |
 
     And the following business categories exist
-      | name         |
-      | Groomer      |
-      | Psychologist |
-      | Trainer      |
+      | name         | subcategories    |
+      | Groomer      | overall grooming |
+      | Psychologist |                  |
+      | Trainer      |                  |
+
+    And the following sub category exist
+      | subcategory_name    | subcategory_description         |
+      | overall grooming    | full service grooming           |
```

Comment:
This is causing test failures because there is no method (in `step_definitions` directory) that implements this step.

I suggest dropping it - we actually don't need the `:description` attribute assignment for subcategories for these particular tests.  (and, if we do need that in the future, we could add a `subcategory_description` field to the existing "the following business categories exist" table).